kahuna/public/js/leases/leases.css

```diff
@@ -4,14 +4,19 @@
     border: 1px solid #999;
     padding: 5px;
     box-sizing: border-box;
+    width: 100%;
```

Comment:
minor: could we [reuse existing](https://github.com/guardian/grid/blob/master/kahuna/public/stylesheets/main.css#L310) classes?